compiler/src/main/java/com/juul/pommel/compiler/ProcessingExtensions.kt

```diff
@@ -0,0 +1,14 @@
+package com.juul.pommel.compiler
+
+import javax.annotation.processing.Messager
+import javax.annotation.processing.RoundEnvironment
+import javax.lang.model.element.Element
+import javax.tools.Diagnostic
+
+/** Return a list of elements annotated with `T`. */
+internal inline fun <reified T : Annotation> RoundEnvironment.findElementsAnnotatedWith(): Set<Element> =
+    getElementsAnnotatedWith(T::class.java)
```

Comment:
Mega nit: I think I'd prefer this if it just used the same function name without any parameters.